src/core/dbus-manager.c

```diff
@@ -2469,19 +2470,23 @@ static int method_disable_unit_files_generic(
         if (r < 0)
                 return install_error(error, r, changes, n_changes);
 
-        return reply_install_changes_and_free(m, message, -1, changes, n_changes, error);
+        return reply_install_changes_and_free(m, message, carries_install_info ? r : -1, changes, n_changes, error);
 }
 
 static int method_disable_unit_files_with_flags(sd_bus_message *message, void *userdata, sd_bus_error *error) {
-        return method_disable_unit_files_generic(message, userdata, unit_file_disable, error);
+        return method_disable_unit_files_generic(message, userdata, unit_file_disable, false, error);
```

Comment:
Nowadays, we add a short comment for boolean argument:
```c
return method_disable_unit_files_generic(message, userdata, unit_file_disable, /* carries_install_info = */ false, error);
```

Reply:
Hmm.. `methos_enable_unit_files_*` are also lack of comments. And I doubt if I should add it to every single one of these functions or only to the first is enough.

Reply:
We did not add such short comment previously. But we recently do. Please.

Reply:
BTW does it make sense to make these functions inline?

Reply:
Hahaha, indeed, that's a good question. If you'd like, please optimize the code. Thank you.

Reply:
It seems that many functions in `dbus-manager.c` require such changes. Would file another PR for those :)

Reply:
Hm, I misunderstood your question, and I thought you want to drop the whole single line functions.
But you suggested to add `inline` for single line functions. But we do not do like that explicitly. Such optimizations should be done by compilers.
Only exception is when the functions are defined in headers.

Reply:
OK, that makes sense.